### ISSUE TITLE
Collimator parameters

### ISSUE THREAD

**Issue by bbazi** [tschoonj/xmimsim#93]
Hi Tom,

I was wondering if the collimator could have some extra parameters to have some more flexibility.
It would be nice if the collimator would have separate input and output diameters, instead of just the input.
In this way we can better simulate collimation where we use multiple pinholes (with different sizes).

Also, it would be very nice to have the fluorescence interact with the collimator material itself, for example in our high-energy experiments we see that the X-rays coming from the sample interact with the collimator inducing extra scatter and fluorescence.
So ideally we would be able to treat the collimator like a sample which has its own density, thickness and list of elements. 
Not sure if it was even planned to update XMI-MSIM any further, these are just some thoughts :-) 

Cheers,
Benjamin

**Comment by bbazi** [comment]
Maybe once my phd is finished, who knows 😄
